samples/drumthumper/README.md

```diff
@@ -24,12 +24,19 @@ Secondarily, **DrumThumper** demonstrates:
 * A mechanism for parsing/loading one type (WAV) of audio data.
 * How to control the relative levels (gain) of audio sources mixed into an output stream.
 * How to locate a mono data source in a stereo output stream.
+* How to use the Oboe resampler to resample source audio to the device playback rate, and therefore not incur this overhead at playback time.
 
 To keep things simple, **DrumThumper** specifically does not:
-* Does not provide support audio samples in other than 16-bit, 44.1K, mono PCM Samples. It does not support Stereo, different samples rates or different PCM formats.
+* Does not provide support audio samples in other than 16-bit, mono PCM Samples. It does not support Stereo or different PCM formats.
 * Does not provide support for non-WAV audio data (such as AIFF).
 * Does not provide support for compressed audio data.
 
+**DrumThumper** does now support different sample rates for the source samples.
```

Comment:
I read this as "does not". Maybe say "now support"

Reply:
"now supports"